File: gs/src/main/java/com/pluxity/device/dto/GsDeviceCctvUpdateRequest.java
package com.pluxity.device.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import java.util.List;

public record GsDeviceCctvUpdateRequest(@Schema(description = "CCTV ID 목록") List<String> cctvIds) {
    public GsDeviceCctvUpdateRequest {
        if (cctvIds == null) {
            cctvIds = List.of();
        }
    }
}

File: gs/src/main/kotlin/com/pluxity/device/dto/GsDeviceCctvUpdateRequest.kt
package com.pluxity.device.dto

import io.swagger.v3.oas.annotations.media.Schema

data class GsDeviceCctvUpdateRequest(
    @field:Schema(description = "CCTV ID 목록")
    val cctvIds: MutableList<String> = mutableListOf(),
)
